mmv1/products/metastore/Federation.yaml

```diff
@@ -144,3 +144,12 @@ properties:
             - 'METASTORE_TYPE_UNSPECIFIED'
             - 'DATAPROC_METASTORE'
             - 'BIGQUERY'
+  - name: 'tags'
+    type: KeyValuePairs
+    description: |
+      A map of resource manager tags.
+      Resource manager tag keys and values have the same definition as resource manager tags.
+      Keys must be in the format tagKeys/{tag_key_id}, and values are in the format tagValues/{tag_value_id}.
+      The field is ignored (both PUT & PATCH) when empty.
```

Comment:
what does it mean that the field is ignored when empty? Especially since this is an immutable field? Possibly this line should just be removed.

```suggestion
```